components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -16,9 +16,12 @@
 
 #import "MDCMultilineTextField.h"
 
+#import "MDCTextField.h"
 #import "MDCTextFieldPositioningDelegate.h"
+#import "MDCTextInputController.h"
 #import "MDCTextInputBorderView.h"
 #import "MDCTextInputCharacterCounter.h"
+#import "MDCTextInputUnderlineView.h"
```

Comment:
Nit: Order of imports. `MDCTextInputController.h` goes after `MDCTextInputCharacterCounter.h`